src/map/MapSidebar.js

```diff
@@ -125,20 +125,26 @@ function MapSidebarMain (props) {
       const currentCeremonyIndex = await api.query.encointerScheduler.currentCeremonyIndex();
       const currentCommunityCeremony = new CommunityCeremony(api.registry, [cid, currentCeremonyIndex]);
 
-      const [assignmentCounts, endorsees] = await Promise.all([
+      const [assignmentCounts, endorsees, tempNewbies] = await Promise.all([
         api.query.encointerCeremonies.assignmentCounts(currentCommunityCeremony),
-        api.query.encointerCeremonies.endorseeCount(currentCommunityCeremony)
+        api.query.encointerCeremonies.endorseeCount(currentCommunityCeremony),
+        api.query.encointerCeremonies.newbieCount(currentCommunityCeremony)
       ]);
+      let newbies;
+      if (currentPhase === 0) {
```

Comment:
no hardcoded indices. should be constant or enum

Reply:
I realized that this is not done nicely in other places in the code, need cleanup later. will merge this

Reply:
should use `.isRegistering` as defined in types